### PR TITLE
Improve/extend description of conditions

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Improves/extends the generated descriptions for conditions in automation.

It takes into account the translations for the attribute names & values from the backend (just like the shown fields do), and does the same for the entity state.

image

image

image

image

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/automation_i18n.ts [hunk 1]
```diff
@@ -513,36 +513,107 @@ export const describeCondition = (
...
+        ? computeStateName(hass.states[condition.entity_id])
+        : condition.entity_id;
+    }
 
+    if (!entities) {
+      // no entity_id or empty array
+      entities = "an entity";
+    }
+
+    base += ` ${entities} is`;
```
Comment (piitaya) [inline src/data/automation_i18n.ts:557]:
```suggestion
    base += ` ${entities} is`;
```
Should we use `are` if these are multiple entities ?

**Review by piitaya — DISMISSED** [review]

**Comment by bdr99** [comment]
I think a comma should only be used if there are three or more items in the list.
 - `A`
 - `A and B`
 - `A, B, and C` (or `A, B and C` if you choose to omit the "Oxford comma" which is just a stylistic choice)

**Comment by frenck** [comment]
Agree; multiple places could do that better. Maybe for a follow up.

File: src/data/automation_i18n.ts (hunk 1 above)
Comment (frenck) [reply src/data/automation_i18n.ts]: image
image

**Review by piitaya — APPROVED** [review]
👍

**Pull request merged**